pkg/config/analysis/analyzers/testdata/service-no-port-name-system-namespace.yaml

```diff
@@ -2,22 +2,6 @@
 # If the service is in system namespace, i.e., kube-system, istio-system, kube-public, the check will be skipped.
 apiVersion: v1
 kind: Service
-metadata:
```

Comment:
I removed these configs since I think we won't apply file configs to namespaces like `kube-system`, and in-cluster configs are already filtered by this PR.

Reply:
If I understand this correctly these would show up if the namespace filtering ever failed/was broken later. If that's the case. it might be best to leave them in to help identify a problem later.

Reply:
@ericvn Thanks for the reminder, fixed.